syntax = "proto3";
import "common.proto";
import "image.proto";
package autodrive.workflow.proto;

message CameraMsg {
  // Header
  MsgHeader header = 1;
  // Error
  Error error = 2;
  // Image
  Image image = 3;
}